frontend/front/src/pages/Public/SurveyPage/SurveyPage.js

```diff
@@ -0,0 +1,123 @@
+import {
```

Comment:
This folder SurveyPage should be deleted since you moved it into pages/SurveyPage